setup.py

```diff
@@ -0,0 +1,99 @@
+"""
+Taken from
+https://github.com/pypa/sampleproject
+"""
+
+# Always prefer setuptools over distutils
+from setuptools import setup, find_packages
+# To use a consistent encoding
+from codecs import open
+import pathlib
+
+
+SETUP_PY_DIRECTORY = pathlib.Path(__file__).resolve().parent
+
+# Get the long description from the README file
+with open(SETUP_PY_DIRECTORY / 'README.md', encoding='utf-8') as f:
+    long_description = f.read()
+
+
+with open(SETUP_PY_DIRECTORY / 'requirements.txt', encoding='utf-8') as f:
+    requirements_file_lines = f.read().strip().splitlines()
+
+
+setup(
+    name='iwant_bot',
+
+    # Versions should comply with PEP440.  For a discussion on single-sourcing
+    # the version across setup.py and the project code, see
+    # https://packaging.python.org/en/latest/single_source_version.html
+    version='0.0.0',
```

Comment:
I don't know how we would like to manage the version number in a smart way - quite honestly, I would like to make a decision after consulting mentors.